README.md

```diff
@@ -1,5 +1,65 @@
 External repository for Bitcoin Core related maintenance tools.
 
+github-merge
+------------
+
+A small script to automate merging pull-requests securely and sign them with GPG.
+
+For example:
+
+```bash
+./github-merge.py 1234
+```
+
+(in any git repository) will help you merge pull request #1234 for the configured repository.
+
+What it does:
+* Fetch master and the pull request.
+* Locally construct a merge commit.
+* Show the diff that merge results in.
+* Ask you to verify the resulting source tree (so you can do a make check or whatever).
+* Ask you whether to GPG sign the merge commit.
+* Ask you whether to push the result upstream.
+
+This means that there are no potential race conditions (where a
+pull request gets updated while you're reviewing it, but before you click
+merge), and when using GPG signatures, that even a compromised GitHub
+couldn't mess with the sources.
+
+Setup
+---------
+Configuring the github-merge tool for the bitcoin repository is done in the following way:
+
+    git config githubmerge.repository bitcoin/bitcoin
+    git config githubmerge.testcmd "make -j4 check" (adapt to whatever you want to use for testing)
+    git config --global user.signingkey mykeyid
```

Comment:
```suggestion
    git config --global user.signingkey mykeyid

If you want to use HTTPS instead of SSH for accessing GitHub, you need set the host additionally:

    git config githubmerge.host "https://github.com"  (default is "git@github.com", which implies SSH)
```

Reply:
Thanks, I've incorporated your change.